src/modules/processing/services/reddit.js

```diff
@@ -48,43 +48,78 @@ async function getAccessToken() {
 }
 
 export default async function(obj) {
-    const url = new URL(`https://www.reddit.com/r/${obj.sub}/comments/${obj.id}.json`);
+    let url = new URL(`https://www.reddit.com/r/${obj.sub}/comments/${obj.id}.json`);
+
+    if (obj.user) {
+        url = new URL(`https://www.reddit.com/user/${obj.user}/comments/${obj.id}.json`);
+    }
 
     const accessToken = await getAccessToken();
     if (accessToken) url.hostname = 'oauth.reddit.com';
 
     let data = await fetch(
-        url, { headers: accessToken && { authorization: `Bearer ${accessToken}` } }
-    ).then((r) => { return r.json() }).catch(() => { return false });
-    if (!data) return { error: 'ErrorCouldntFetch' };
+        url, {
+            headers: {
+                authorization: `Bearer ${accessToken}`,
```

Comment:
the `accessToken && ...` is needed, ``` `Bearer ${accessToken}` ``` sends `'Bearer undefined'` if there is no accessToken

Reply:
(and it breaks reddit downloads for instances with no cookies)